ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.10.yaml

```diff
@@ -65,19 +69,111 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: gcp-ipi-f14
-  cron: 5 13 8,24 * *
+- as: aws-ipi-proxy-sts-f14
```

Comment:
f28?

Reply:
The job is intended to test sts. If it's acceptable to test sts once a month, then I'm fine with f28.

Reply:
I am thinking we are adding more coverage in the chain upgrade, maybe in the future, we would add more, to avoid overload, I would suggest `f28`. cc @yunjiang29

Reply:
Understand your concern.
For the 4.12+, we have one z release per week. For those important configurations, I'm hoping we test them once per week in chain upgrade at least. So the frequency depends on how important sts is. @yunjiang29 we need your expertise.
In the future, if a new release branch is forked, we need to review the chain upgrade test matrix/frequency and revise the jobs accordingly.

Reply:
I'm good with `14`, we can adjust frequency if more sts profiles are added.